src/main/java/it/pagopa/ecommerce/commons/exceptions/CheckoutRedirectMissingPspRequestedException.java

```diff
@@ -0,0 +1,26 @@
+package it.pagopa.ecommerce.commons.exceptions;
+
+import java.util.Set;
+
+/**
+ * Exception thrown when requesting an API key from NPG configuration for a
+ * nonexisting PSP
+ */
+public class CheckoutRedirectMissingPspRequestedException extends RuntimeException {
```

Comment:
same as above, this exception can cover also other configuration missing cases, such as missing logo or b.e. url

Reply:
Fixed with f103dcb8